src/test/resources/appium/inAppBidding_requests/ios/original_api/inApp_auction_320x480_original.json

```diff
@@ -0,0 +1,125 @@
+{
+  "app": {
+    "ext": {
+      "data":{
+
+      },
+      "prebid": {
+        "source": "prebid-mobile",
+        "version": "$VARIABLE"
+      }
+    },
+    "ver": "$VARIABLE",
+    "name": "InternalTestApp",
+    "publisher": {
+      "id": "$VARIABLE"
+    },
+    "bundle": "$VARIABLE"
+  },
+  "ext": {
+    "prebid": {
+      "storedrequest": {
+        "id": "$VARIABLE"
+      },"storedauctionresponse": {
+        "id": "$VARIABLE"
+      },
+      "cache": {
+        "bids": {
+
+        },
+        "vastxml": {
+
+        }
+      },
+      "targeting": {
+
+      },
+      "data": {
+        "bidders": []
+      }
+    }
+  },
+  "id": "$VARIABLE",
+  "regs":{
+    "ext":{
+      "gdpr":"$VARIABLE"
+    }
+  },
+  "source": {
+    "ext": {
+      "omidpv": "$VARIABLE",
+      "omidpn": "Prebid"
+    },
+    "tid": "$VARIABLE"
+  },
+  "device": {
+    "ext": {
+      "atts": 0,
+      "ifv": "$VARIABLE",
+      "prebid": {
+        "interstitial": {
+          "minheightperc": 70,
+          "minwidthperc": 50
+        }
+      }
+    },
+    "os": "iOS",
+    "ifa": "$VARIABLE",
+    "h": "$VARIABLE",
+    "language": "$VARIABLE",
+    "ua": "$VARIABLE",
+    "geo": {
+
+    },
+    "pxratio": "$VARIABLE",
+    "lmt": "$VARIABLE",
+    "osv": "$VARIABLE",
+    "w": "$VARIABLE",
+    "model": "iPhone",
+    "connectiontype": "$VARIABLE",
+    "make": "Apple"
+  },
+  "imp": [
+    {
+      "ext": {
+        "context": {
+          "data": {
+
+          }
+        },
+        "prebid": {
+          "storedrequest": {
+            "id": "$VARIABLE"
+          },"storedauctionresponse": {
+            "id": "$VARIABLE"
+          }
+        }
+      },
+      "clickbrowser": 0,
+      "displaymanager": "prebid-mobile",
+      "displaymanagerver": "$VARIABLE",
+      "banner": {
+        "format": [
+          {
+            "w": "$VARIABLE",
```

Comment:
Since we know the test device, we should know its size. Let's use particular values for all templates in this PR instead of `$VARIABLE`.